portfolio/src/main/java/com/google/sps/servlets/GameServlet.java

```diff
@@ -34,7 +34,7 @@
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        game.generateFacts();
+        game.generateFacts(); //facts received from server
```

Comment:
nit: I don't think this comment adds much. The reader can interpret that from reading the code.

Reply:
Fixed!